context/brand-context/HISTORY.md

```diff
@@ -1,5 +1,9 @@
 # History
 
+## 10.0.5 (2021-02-23)
```

Comment:
This should be `10.1.0` as it includes a new feature.
Add a comment to specify the addition of `@mixin u-heading` for heading switching